torch/csrc/jit/passes/onnx/peephole.cpp

```diff
@@ -661,6 +661,46 @@ static void fuseUnbindListUnpack(Block *b) {
   }
 }
 
+// Traced Split with list of sizes is being converted to ONNX as SplitToSequence + SequenceAt.
+// Example IR
+//  %2 : int = prim::Constant[value=0]
+//  %3 : Tensor[] = aten::split_with_sizes(%input, %split_list, %2)
```

Comment:
Maybe we should change the comment here to onnx::SplitToSequence? Since the pass is changing SplitToSequence + ListUnpack to SplitToSequence + SequenceAt

Reply:
Thanks, updated that.